Update maia-chat.md (#9002)
mendix/docs: content/en/docs/refguide/modeling/mendix-ai-assistance/maia-chat.md

## Introduction 

{{% alert color="info" %}}

> MendixChat was rebranded to Maia Chat and released in GA in Studio Pro 10.12.0 and the Mendix Portal.

Reviewer: ```suggestion
MendixChat was rebranded to Maia Chat and released in GA in Studio Pro 10.12.0. It is also available in the Mendix Portal.
```

{{% /alert %}}

{{% alert color="info" %}}

> To use Maia Chat, internet connection and signing in to Studio Pro or the Mendix Portal are required.

Reviewer: ```suggestion
To use Maia Chat, an internet connection and signing in to Studio Pro or the Mendix Portal are required.
```

> To use Maia Chat, an internet connection and signing in to Studio Pro or the Mendix Portal are required.

Reviewer: ```suggestion
To use Maia Chat, an internet connection and signing in to Studio Pro are required.
```

{{% /alert %}}

{{% alert color="info" %}}
Maia Chat does not use any project, customer, or company information. It only uses the data that you have entered in the Chat. Please do not enter any personal or sensitive data!
{{% /alert %}}

> Maia Chat is a built-in chat interface powered by Generative AI in the Mendix Platform . Developers can ask questions about app development in Mendix, including how to apply concepts, best practices, and development patterns. It is built based on large language models (LLMs), using data from various resources, such as [Mendix Docs](/), [Mendix Community](https://community.mendix.com/), and [Mendix Academy](https://academy.mendix.com/).

Reviewer: ```suggestion
Maia Chat is a built-in chat interface in the Mendix Platform powered by Generative AI. Developers can ask questions about all aspects of Mendix, including how to apply concepts, best practices, governance tools, and development patterns. It is based on large language models (LLMs) using data from various resources, such as [Mendix Docs](/), [Mendix Community](https://community.mendix.com/), and [Mendix Academy](https://academy.mendix.com/).
```

## Using Maia Chat

There are two ways to access Maia Chat: 

> * Studio Pro and Mendix Portal: You can click the **Maia** ({{% icon name="sparkles" %}}) icon at the top right bar.
> * Studio Pro: Or you can go to the **View** menu and click **Maia**

Reviewer: ```suggestion
* In Studio Pro and Mendix Portal, click the **Maia** ({{% icon name="sparkles" %}}) icon to the right of the top bar.
* In Studio Pro go to the **View** menu and click **Maia**
```

> The chat interface appears at the right side of your context window or browser. In Studio Pro you can find the dock under the **Maia** tab.

Reviewer: What does "context window" mean?

    * An answer generated by Maia Chat
    * Top relevant references from various resources like Mendix documentation, Mendix Community, and Academy learning paths

3. Click the thumbs up icon ({{% icon name="thumbs-up" %}}) or the thumbs down icon ({{% icon name="thumbs-down" %}}) to give your feedback on an answer. 

4. Click the add feedback icon ({{% icon name="notes-paper-edit" %}}) to provide a more detailed feedback. A detailed **Feedback** section shows below the answer:

    * You can rate the **correctness**, **completeness**, and **readability** of the provided answer.

    * You can also provide **Additional feedback** if you want to. 

      {{< figure src="/attachments/refguide/modeling/mendix-ai-assistance/maia-chat/feedback-section.png" width="250px" >}} 

5. Click the **Submit feedback** button to submit your feedback. Afterwards, you will be brought back to the main chat interface where you can continue asking questions.

6. Click **New chat** at beginning of your chat if you want to clear the messages and start a new conversation without reference to your current chat (this option was added in Studio Pro 10.7.0).
#### Studio Pro Features

> Maia Chat in Studio Pro has additional features which are indicated by the following:

Reviewer: I know you still need to tidy this up - just making a note here!

* A copy icon ({{% icon name="copy" %}}) which allows you to copy the given answer to a clipboard (this option was added in 10.7.0)
* Feedback icons
* New chat option
  
    {{< figure src="/attachments/refguide/modeling/mendix-ai-assistance/maia-chat/maia-chat-example.png" width="350px" >}}
    
    {{% alert color="info" %}}In Studio Pro 10.7.0 and above, you can also see the Mendix version of the references, the level required if the reference comes from Mendix Academy, and the date and answer status if the reference comes from the Mendix Community.{{% /alert %}}

### Asking Follow-Up Questions

In Studio Pro 10.8.0 and above, Maia Chat supports asking follow-up questions based on your previous question and the answer you have received. For example, you can ask it to summarize the latest answer you received by typing the following in the chat: *Can you summarize that?*.

{{% alert color="info" %}}

> Maia Chat only keeps track of a conversation history within a session in Studio Pro.

Reviewer: Do we need to be clearer about this. I don't think we are talking about an active session (i.e. Studio Pro remembering who we are) in the way we normally talk about sessions in app development. So I would probably say something like

```suggestion
Maia Chat will lose track of the conversation history if you exit Studio Pro.
```

1. Type the following question in the chatbox: `When should I use a calculated attribute?`.
2. Press <kbd>Enter</kbd> or the **Ask** button to send the question. You will then see the following:
    * An answer generated by Maia Chat

>     * Studio Pro: A copy icon ({{% icon name="copy" %}}) which allows you to copy the given answer to a clipboard (this option was added in 10.7.0)
>     * Studio Pro: Feedback icons
>     * Top relevant references from various resources like Mendix documentation, Mendix Community, and Academy learning paths

Reviewer: ```suggestion
    * Top relevant references from various resources like Mendix documentation, Mendix Community, and Academy learning paths
    
#### Studio Pro Features
    
Studio Pro has additional features which are indicated by the following:
    
    * A copy icon ({{% icon name="copy" %}}) which allows you to copy the given answer to a clipboard (this option was added in 10.7.0)
    * Feedback icons
    * New chat option
```

> 3. To give your feedback on the given answer, click the thumbs up icon ({{% icon name="thumbs-up" %}}) or the thumbs down icon ({{% icon name="thumbs-down" %}}). 

Reviewer: ```suggestion
1. Click the thumbs up icon ({{% icon name="thumbs-up" %}}) or the thumbs down icon ({{% icon name="thumbs-down" %}}) to give your feedback on an answer. 
```

> 4. In Studio Pro: To provide a more detailed feedback, click the add feedback icon ({{% icon name="notes-paper-edit" %}}). A detailed **Feedback** section shows below the given answer:

Reviewer: ```suggestion
1. Click the add feedback icon ({{% icon name="notes-paper-edit" %}}) to provide a more detailed feedback. A detailed **Feedback** section shows below the answer:
```

    * You can rate the **correctness**, **completeness**, and **readability** of the provided answer.

>     * You can add additional explanation text in the **Additional feedback** text box if you want to. 

Reviewer: ```suggestion
    * You can also provide **Additional feedback** if you want to. 
```

        {{< figure src="/attachments/refguide/modeling/mendix-ai-assistance/maia-chat/feedback-section.png" width="250px" >}} 

> 5. In Studio Pro: Click the **Submit feedback** button to send your feedback. Afterwards, you will be brought back to the main chat interface where you can continue asking new questions.

Reviewer: ```suggestion
1. Click the **Submit feedback** button to submit your feedback. Afterwards, you will be brought back to the main chat interface where you can continue asking questions.
```

> 6. In Studio Pro: If you want to clear the messages to start a new conversation in a clean state, navigate to the top of the interface and click **New chat** (this option was added in 10.7.0).

Reviewer: ```suggestion
1. Click **New chat** at beginning of your chat if you want to clear the messages and start a new conversation without reference to your current chat (this option was added in 10.7.0).
```

Reviewer: Was this really added in 10.7.0, two lines down we say that follow-up questions were only introduced in 10.8.0, so the **New Chat** options wouldn't have been necessary.

### Asking Follow-Up Questions

> In Studio Pro 10.8.0 and above and the Mendix Portal, Maia Chat supports asking follow-up questions based on your previous question and the answer you have received. For example, you can ask it to summarize the latest answer you received by typing the following in the chat: *Can you summarize that?*.

Reviewer: ```suggestion
In Studio Pro 10.8.0 and above, and in Mendix Portal, Maia Chat supports asking follow-up questions based on your previous question and the answer you have received. For example, you can ask it to summarize the latest answer you received by typing the following in the chat: *Can you summarize that?*.
```

> ### Conversation History

Reviewer: I think this should be a subsection of **Asking Follow-Up Questions** as it doesn't make sense unless Maia knows about your previous chat.
```suggestion
#### Conversation History

```
